.github/workflows/check-spelling.yml

```diff
@@ -0,0 +1,14 @@
+name: Check spelling # cSpell:ignore refcache
+
+on:
+  pull_request:
+
+jobs:
+  check-formatting:
+    name: Check spelling
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      # cspell has no dependencies that need to be installed so we can skip this step
+      # - run: npm install
```

Comment:
Optional, but speeds up the whole action by ~50% (from above 1 minute down to ~30s)